src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -517,6 +517,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CustomLinkerArg Condition="'$(Optimize)' != 'true'" Include="$(WasmOptimizationSetting) -flto" />
       <CustomLinkerArg Condition="'$(IlcLlvmTarget)' != ''" Include="-target $(IlcLlvmTarget)" />
       <CustomLinkerArg Condition="'$(IlcLlvmExceptionHandlingModel)' == 'wasm'" Include="-fwasm-exceptions" />
+      <CustomLinkerArg Condition="'$(StackTraceSupport)' == 'false'" Include="-Wl,--strip-all" />
```

Comment:
Hmm, this will strip all debug info, not just the stack trace data, which is not very satisfactory. Let's bundle this work into #2404 and disable the tests.